packages/core/content-releases/admin/src/pages/ReleasesPage.tsx

```diff
@@ -283,9 +309,42 @@ const ReleasesPage = () => {
   };
 
   return (
-    <ReleasesLayout onClickAddRelease={toggleAddReleaseModal} totalReleases={totalReleases}>
+    <ReleasesLayout
+      onClickAddRelease={toggleAddReleaseModal}
+      totalReleases={totalReleases}
+      isPrimaryActionDisabled={hasReachedMaximumPendingReleases}
+    >
       <ContentLayout>
         <>
+          {activeTab === 'pending' && hasReachedMaximumPendingReleases && (
+            <CustomAlert
```

Comment:
nitpick/observation not a request for change per se: I notice sometimes in the codebase styled components use either a prefix Custom or a prefix Styled. I think a convention that is trending right now in the codebase is to refer to styled components with the Styled prefix instead of the Custom prefix. I think it would be nice if we choose one and we are consistent. You don't have to change this though if you don't want.